tests/scripts/config.sh.example

```diff
@@ -1,8 +1,10 @@
+#!/bin/sh
```

Comment:
Nope.
This file is meant to be sourced.
It should not be executable to begin with. And hence no shebang.